front/public/locales/de/translation.json

```diff
@@ -73,7 +73,8 @@
       "de": "Deutsch",
       "en": "English",
       "fr": "Français",
```

Comment:
Side note: since these are never translated, should we really put them in translation files?

Reply:
Mmm maybe not